src/app/(main)/connections/_components/connection-table.tsx

```diff
@@ -89,13 +89,17 @@ export default function ConnectionTableServer() {
           <TableBody>
             {connections.map((conn) => (
               <TableRow key={conn.name}>
-                {["name", "host", "username", "password", " dc", "nodes"].map(
+                {["name", "host", "username", "password", "dc", "nodes"].map(
                   (key) => (
                     <ContextMenu key={`${conn.name}-${key}`}>
                       <ContextMenuTrigger asChild>
                         <TableCell>
-                          {key === "password" ? "••••••••" : (conn as any)[key]}
-                        </TableCell>
+                          {key === "password"
+                            ? "••••••••"
+                            : key === "dc" // since dc will be retrived from the driver it is hardcoded for now
```

Comment:
Instead hardcoded it here, you can just add directly into the INSERT query. Furthermore we would be tweaking there to get these missing numbers.

Any Database with Partition Tolerance would have 1 or more nodes, so you can just expect it.

TLDR: remove this rule and add the `3` hardcdoed to the database.